zsh-aws-vault.plugin.zsh

```diff
@@ -48,6 +48,14 @@ function avli() {
         ;;
     esac
 
+  elif (command -v "google-chrome" > /dev/null); then
```

Comment:
I'd love to see this block follow the pattern above, e.g.

```
elif _using_linux ; then
  case $browser in
     google-chrome)
         #your logic
         ;;
   esac
```

That should be pretty straightforward to do, by adding a `uname` check for `Linux` and adjusting the `_find_browser` to have logic for Linux (using the `command -v` logic you have would be perfect). It's totally fine if you only implement the `google-chrome` logic and leave the rest for others who are willing to PR the project for Firefox, etc.